omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -121,10 +122,11 @@
     location ~ <%= @compliance_proxy_regex -%> {
       set $request_org $1;
       access_by_lua_block { validator.validate("GET") }
+      proxy_set_header Host $proxy_host;
       proxy_set_header x-data-collector-token $data_collector_token;
       proxy_set_header x-data-collector-auth "version=1.0";
       rewrite ^<%= @compliance_proxy_regex -%> /compliance/profiles/$2$3 break;
-      proxy_pass https://compliance-profiles;
+      proxy_pass <%= URI.parse(node['private_chef']['profiles']['root_url']).scheme %>://<%= URI.parse(node['private_chef']['profiles']['root_url']).host %>:<%= URI.parse(node['private_chef']['profiles']['root_url']).port %>;
```

Comment:
Similarly is there a `compliance-profiles` location that we can remove with this change?